test/Board-test.js

```diff
@@ -10,13 +10,51 @@ const Board = require('../lib/Board.js');
 // chai.spy.on(global.domUpdates, [], () => true); 
 
 describe('Board', () => {
-let board; 
+  let board; 
 
-beforeEach(() => {
-    board = new Board()
-})
+  beforeEach(() => {
+    let puzzleObject = {  
+      category: 'Around The House',
+      number_of_words: 1,
+      total_number_of_letters: 8,
+      first_word: 8, 
+      description: 'Location or object(s) found within a typical house.',
+      correct_answer: 'Armchair',
+    }
+    board = new Board(puzzleObject.category, puzzleObject.correct_answer);
+  })
 
-it('should instantiate a new Board class', () => {
+  it('should instantiate a new Board class', () => {
     expect(board).to.be.an.instanceof(Board); 
-})
+  })
+
+  it('should have properties of correctAnswer and category from passed in arguments', () => {
+    expect(board.correctAnswer).to.equal('Armchair');
+    expect(board.category).to.equal('Around The House');
+  })
+
+  it('should have properties of rows one through four and correctAnswerArray that are empty by default', () => {
+    expect(board.row1).to.eql([]);
+    expect(board.row2).to.eql([]);
+    expect(board.row3).to.eql([]);
+    expect(board.row4).to.eql([]);
+    expect(board.correctAnswerArray).to.eql([]);
+  })
+
+  it('should have a method to update the category on the DOM', () => {
+    //chai spies for call to method
+  })
+
+  it('should have a method to display the letter guessed on the DOM', () => {
+    //chai spies
+  })
+
+  it('should have a method to display the correct answer on the DOM', () => {
+    // chai spies
+  })
+
+  it('should be able to split the correct answer into an array of letters', () => {
+    board.splitAnswer();
```

Comment:
Mike great work on all of these test, this is an amazing job. One suggestion, on line 57 we would want to pass in a string, to represent the answer we want to split.

Reply:
now that i look at it, the method uses the correct answer property of the board class to split, so it doesn't need to have anything passed in. Should we change that?